analysis/preregistration_template.Rmd

```diff
@@ -194,83 +219,90 @@ To be honest, I don't know how I might apply this to my work or any future works
 
 ## 6.1 Choose performance criteria for scenario analysis
-
+HF- I feel that this belongs at the end of section 1
 *Describe how you will evaluate and distinguish the performance of alternative scenario outcomes. These measures should link back to the analytical objectives specified during the problem formulation phase, and might include for example, descriptive statistic measures, such as a model in the top 10th percentile of worst conditions, satisficing measures such as a minimum performance threshold for achieving some management objective, or perhaps regret measures, such as choosing a decision with minimum regret [@Moallemi2019]."Justify or otherwise explain how you chose these measures, and determined performance criteria in relation to the analytical objectives, modelling context, such as the risk attitudes of decision-makers and stakeholders within this system.*
 
-## 6.2 Synthesis of model application results	
 
+## 6.2 Synthesis of model application results	
+HF- I think this should stay here at the end
 *Specify how you will map, plot or otherwise display and synthesise the results of scenario and model analyses. Explain how will they support meaningful analysis and interpretation to aid in answering the analytical objectives and modelling context specified in the problem formulation phase.Explain how you will analyse the model analysis output to answer your analytical objectives. For instance, describe any trade-off or robustness analyses you will undertake to help evaluate and choose between different alternatives in consultation with experts or decision-makers.*
 
 ## 6.3 Consultation with Experts, Clients, Stakeholders	
-
-*How will you use the results of model application analysis to provide advice. and support to the clients / managers / stakeholders based on these findings?Describe how these results will be characterised and presented to experts, clients and stakeholders once the results from the modelling exercises have been synthesised. What role will these consultations have in determining the selection of management interventions, or future iterations of model development? For example, what key outliers or strongest examples of best or worst case scenarios should be presented to experts or decision-makers?(Do we need an item about how do we interpret the model outputs? Should there be some a priori decisions about how we interpret the model findings, and how does case 49 about the risk attitude inform how the analyst interprets the model findings?)*
+HF- I'm not sure how possible people will find doing this... and whether it is potentially problematic to ask people to because their clients will amost certainly change their mind about how they deal with the information at this point in a way that the researcher cant control. 
```

Comment:
Yeah. I wasn't sure whether to include this, or just acknowledge this as being a part of the modelling workflow for people working in government institutions and agencies on decision-problems and then removing the item from the template.

I will leave this in for now, but set up an issue to invite @csjones8 to the conversation.

As an explanation for why I included this item, basically all people in the modelling group included a decision-point item about checking back with clients or others at the end.

You're right, it could put the researcher in a tricky position, however, I'm not sure. Perhaps when I send the survey out to workshop participants we will get a good sense of how people feel about this.

I do feel that it's important to be transparent about when evidence for a particular management intervention arising from a structured decision analysis will be actually based on that analysis, or whether that exercise will be ignored in favour of clients who might be locked onto a particular candidate decision, rather than some ecological outcome.